src/applications/find-forms/helpers/index.js

```diff
@@ -95,3 +95,13 @@ export const setCookie = () => {
     document.cookie = `findForms=pdfModal; expires=${utcDateString};`;
   }
 };
+
+export const deriveDefaultModalState = () => {
```

Comment:
```js
export const deriveDefaultModalState = () => ({
  isOpen: false,
  pdfSelected: '',
  pdfUrl: '',
  pdfLabel: '',
  doesCookieExist: false,
});
```

You can do this if you want 🙂

Reply:
ahh true, I like the explicit return statement for readability, but i appreciate it!